pkg/cluster/describe.go

```diff
@@ -158,15 +159,30 @@ func PrintClusterDescription(connection *sdk.Connection, cluster *cmv1.Cluster)
 	}
 
 	// Check Hypershift-related values
-	mgmtCluster := ""
+	mgmtClusterId := ""
+	svcClusterId := ""
 	if cluster.Hypershift().Enabled() {
-		// Ignorning the error here as this endpoint is behind a specific permissioon
+		// Ignoring errors here as this endpoint is behind specific permissions
 		hypershiftResp, _ := connection.ClustersMgmt().V1().Clusters().
 			Cluster(cluster.ID()).
 			Hypershift().
 			Get().
 			Send()
-		mgmtCluster = hypershiftResp.Body().ManagementCluster()
+		fmMgmtResp, _ := connection.OSDFleetMgmt().V1().ManagementClusters().
+			List().
+			Parameter("name", hypershiftResp.Body().ManagementCluster()).
+			Send()
+
+		mgmtClusterId = fmMgmtResp.Items().Get(0).ClusterManagementReference().ClusterId()
+
+		// href is in the format /api/osd_fleet_mgmt/v1/service_clusters/${id}
+		// and we are using it because ocm-sdk-go currently does not export the id directly
+		if kind := fmMgmtResp.Items().Get(0).Parent().Kind(); kind == "ServiceCluster" {
+			href := fmMgmtResp.Items().Get(0).Parent().Href()
+			_, id := path.Split(href)
+			fmSvcResp, _ := connection.OSDFleetMgmt().V1().ServiceClusters().ServiceCluster(id).Get().Send()
+			svcClusterId = fmSvcResp.Body().ClusterManagementReference().ClusterId()
```

Comment:
The main reason I switched the management cluster to show its internal ID instead of the name as it was previously was due to the cluster name of the service cluster not being available here - I'd have to make an additional call to ClustersMgmt in order to get the name of a service cluster.

Happy to do so if that's preferred, but I felt that it was ok to show the ID instead since those are guaranteed to be unique.

cc @fahlmant

Reply:
Either way works, this is for the SRE mostly, so its up to us to decide. We're already making multiple API calls, so I don't think one more would hurt, and reasoning about a name like `hs-mc-xxxxx` is a bit easier than an ID, but we use both constantly and internal ID is unique.

tl;dr either way works, there's an argument for both ways being a better UX so I have no idea what we should do

Reply:
Got it - yeah agreed that `hs-mc-xxxx`/`hs-sc-xxxx` is easier to grok if this is mostly for SRE. I'll make that change

Reply:
Is there anyway to enforce this only shows up for specific roles/permissions taken from token? Don't think usual clients will get much from this info

Reply:
During printing, we already do this check 
```
if svcClusterId != "" {
```

Reply:
also https://github.com/openshift-online/ocm-cli/pull/441/files#diff-2f84bc16b5d89dac13d11d6f8ae66c242bc75cccb194fdd0ace7723bb75b9312R165 we ignore errors and don't print anything by default - so it will only print if the specific roles/permissions are in place